src/frontend-pwa/src/App.tsx

```diff
@@ -8,20 +8,21 @@ import SplashScreen from './components/SplashScreen/SplashScreen';
 
 function App() {
   const [isLoading, setIsLoading] = useState(true);
+  const [locationData, setLocationData] = useState([]);
```

Comment:
Let's type check this:

`useState<LocationProps>(defaultData)`

Reply:
@Bottle7 It is done!